pkgs/applications/networking/instant-messengers/ripcord/default.nix

```diff
@@ -6,11 +6,12 @@
 mkDerivation rec {
   pname = "ripcord";
   version = "0.4.24";
+  imageHash = "0rscmnwxvbdl0vfx1pz7x5gxs9qsjk905zmcad4f330j5l5m227z";
 
   src = let
     appimage = fetchurl {
       url = "https://cancel.fm/dl/Ripcord-${version}-x86_64.AppImage";
-      sha256 = "0rscmnwxvbdl0vfx1pz7x5gxs9qsjk905zmcad4f330j5l5m227z";
+      sha256 = "${imageHash}";
```

Comment:
This doesn't do what you expect it to. Overriding both `version` and `imageHash` will still result in it using the old source

Reply:
oh, gotcha. How best then?

Reply:
override the `src` entirely

Reply:
Figures. I'll scratch this then.